lib/v2/zimuxia/portfolio.js

```diff
@@ -17,16 +18,13 @@ module.exports = async (ctx) => {
     const $ = cheerio.load(response.data);
 
     const items = $('a')
-        .filter((_, el) => $(el).attr('href')?.startsWith('magnet:'))
+        .filter((_, el) => $(el).attr('href')?.match(RegExp(linktype)))
```

Comment:
GHSA-jvxx-v45p-v5vf

Reply:
Thanks for the remind. I limited linktype regex to a list of choice, this would not allow user regex thus would solve this problem? [c1dd995](https://github.com/DIYgod/RSSHub/pull/13783/commits/c1dd99580316ec990c937da2036cbe6ccbeabd6b)